sdk/spring-sdk/src/main/java/kalix/springsdk/annotations/Publish.java

```diff
@@ -32,4 +32,26 @@
     /** Assign the name of the topic to be used for eventing out. */
     String value();
   }
+
+
+  @Target(ElementType.TYPE)
+  @Retention(RetentionPolicy.RUNTIME)
+  @Documented
+  @interface EventStream {
+
+    String name();
+
+    /**
+     * Assign the class type of the entity one intends to subscribe to, which must extend {@link
+     * kalix.javasdk.eventsourcedentity.EventSourcedEntity EventSourcedEntity}.
+     */
+    Class<? extends kalix.javasdk.eventsourcedentity.EventSourcedEntity<?>> source();
```

Comment:
What about VE?

Reply:
no supported yet
 
and then, it should not be on `@Publish.EventStream`, but something else, eg: `@Publish.ChangeStream' or a better name

Reply:
Ok yeah, so I was looking at this from the second option perspective (where we have to separate annotations). In that sense, it would be similar to `@Publish.Topic` where it's not typed as well... Being all in one, it makes sense to be typed.

Reply:
I split the annotations so the discussion here is not relevant anymore. We can revisit it with we take that path in the future.